aldryn_jobs/models.py

```diff
@@ -194,40 +197,33 @@ def get_absolute_url(self, language=None):
     def get_notification_emails(self):
         return self.supervisors.values_list('email', flat=True)
 
+
 @version_controlled_content(follow=['category', 'app_config'])
+@python_2_unicode_compatible
 class JobOffer(TranslatableModel):
     translations = TranslatedFields(
         title=models.CharField(_('Title'), max_length=255),
-        slug=models.SlugField(
-            _('Slug'), max_length=255, blank=True,
+        slug=models.SlugField(_('Slug'), max_length=255, blank=True,
```

Comment:
For "normal" field definitions, yes, but this is actually a parameter for the translations() method. PEP8 frowns upon spaces here.